examples/rag_pipeline.py

```diff
@@ -0,0 +1,248 @@
+"""RAG pipeline"""
+
+import asyncio
+
+from pydantic import BaseModel
+
+from metagpt.const import DATA_PATH, EXAMPLE_DATA_PATH
```

Comment:
You should merge or rebase from the existed code branch, not just add the files as new files. The commit history will break.